server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -895,19 +901,20 @@ private List<String> createClusterChain(final Map<String, ClusterMetadataManifes
                         Locale.ROOT,
                         "The system has ended into multiple valid cluster states in the remote store. "
                             + "Please check their latest manifest to decide which one you want to keep. Valid Cluster UUIDs: - %s",
-                        validClusterUUIDs
+                        topLevelClusterUUIDs
                     )
                 );
             }
             return createClusterChain(manifestsByClusterUUIDTrimmed, clusterName);
         }
         final List<String> validChain = new ArrayList<>();
-        String currentUUID = validClusterUUIDs.get(0);
+        String currentUUID = topLevelClusterUUIDs.get(0);
         while (currentUUID != null && !ClusterState.UNKNOWN_UUID.equals(currentUUID)) {
             validChain.add(currentUUID);
             // Getting the previous cluster UUID of a cluster UUID from the clusterUUID Graph
             currentUUID = clusterUUIDGraph.get(currentUUID);
         }
+        logger.info("Constructed cluster chain as : {}", validChain);
```

Comment:
[this PR](https://github.com/opensearch-project/OpenSearch/pull/10892) also added a log here - once you rebase you can make that log as info

Reply:
Sure. Will update once the mentioned PR is merged.